hedera-mirror-rest/balances.js

```diff
@@ -158,10 +157,16 @@ const getBalances = async (req, res) => {
 };
 
 const getAccountBalanceTimestamp = async (tsQuery, tsParams, order = 'desc') => {
+  if (!tsQuery || !tsQuery.includes('account_id')) {
+    // Add the treasury account to the query as it will always be in the balance snapshot and account_id is the primary key of the table thus it will speed up queries on v2
+    tsQuery = tsQuery ? tsQuery.concat(' and account_id = ?') : ' account_id = ?';
+    tsParams.push('2');
+  }
```

Comment:
`tsQuery` built by the caller shouldn't have any `account_id = ` query.
  
```suggestion
  // Add the treasury account to the query as it will always be in the balance snapshot and account_id is the primary key of the table thus it will speed up queries on v2
  tsQuery = tsQuery ? tsQuery.concat(' and account_id = ?') : ' account_id = ?';
  tsParams.push('2');
```

Reply:
Got it, I have removed the if statement